lib/Controllers/list.js

```diff
@@ -106,6 +102,15 @@ List.prototype.fetch = function(req, res, context) {
       var end = start + result.rows.length - 1;
       end = end === -1 ? 0 : end;
 
+      if (options.include) {
```

Comment:
All this does is remove the `multiassoction_ids` param from the dataset. I don't see any reason somebody would want to change that. You're still able manipulate the `multiassocitioan: [record1, record2, etc]` object in the `before.send`.